### PR TITLE
Import Firefox 68 schema.

### LINKED ISSUE #2623: Import Firefox 68 schema
There is already a `FIREFOX_68_0b6_RELEASE` tag so we should import the schema as soon as possible.

### PR DESCRIPTION
**Pull request by EnTeQuAk**
Fixes #2623

### CODE REVIEW

File: src/schema/imported/index.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import bookmarks from './bookmarks.json';
 import browserAction from './browser_action.json';
 import browserSettings from './browser_settings.json';
 import browsingData from './browsing_data.json';
+import captivePortal from './captive_portal.json';
```
Comment (muffinresearch) [inline src/schema/imported/index.js]: The tests are failing on not being able to find this by the looks of it.

Comment (EnTeQuAk) [reply src/schema/imported/index.js]: Indeed, I pushed the missing file in 2d7ccde3. Totally forgot :disappointed:

**Comment by muffinresearch** [comment]
Tests are passing, it looks sane - I'm not sure otherwise how to review this as I'm not familiar enough with the incoming changes.

Could you ask QA to ensure they throw plenty of real add-ons through validation to this in action?

r+wc

**Comment by EnTeQuAk** [comment]
> Could you ask QA to ensure they throw plenty of real add-ons through validation to this in action?

Absolutely

**Pull request merged**